### PR TITLE
Report overproduction fix

### PR DESCRIPTION
**Pull request by maledo**

### CODE REVIEW

File: urbs.py [hunk 1]
```diff
@@ -1894,18 +1895,27 @@ def report(instance, filename, commodities=None, sites=None):
...
                 overprod = pd.DataFrame(
                     columns=['Overproduction'],
                     data=created.sum(axis=1) - consumed.sum(axis=1) +
                     imported.sum(axis=1) - exported.sum(axis=1) +
                     stored['Retrieved'] - stored['Stored'])
-
+                
+                # add timeseries demand and delta again to consumed
+                consumed = pd.concat(
+                    (consumed, demand.rename('Demand'),
```
Comment (ojdo) [inline urbs.py]: Why is there a rename? Both Series seem to have the same name as when pop'ed in lines 1900, 1901.

File: urbs.py [hunk 2]
```diff
@@ -1894,18 +1895,27 @@ def report(instance, filename, commodities=None, sites=None):
             for sit in sites:
                 created, consumed, stored, imported, exported, derivative = get_timeseries(
                     instance, co, sit)
+                
+                # temporarily drop out the timeseries demand and delta, for a correct overproduction calculation
```
Comment (ojdo) [inline urbs.py]: I'm a bit unsure whether this approach "scales" well. In my books, the expression that calculates `overprod` should always hold, not only after removing two specially named columns. In other words: the uninitiated observer looking at the resulting report spreadsheet has no way of inferring - from the visible structure alone - how the column overproduction was calculated. He/she must instead analyse the meanings of all columns and derive that two (demand, demand-delta) columns are special and not part of the balance.

File: urbs.py (hunk 1 above)
Comment (maledo) [inline urbs.py]: When you pop, the series has no name. So to set the name, the rename function is used.

File: urbs.py (hunk 2 above)
Comment (maledo) [inline urbs.py]: These two columns always exists at the moment. At first I was trying to just add the columns needed (Feed-in and Demand/Shifted Demand). But Feed-in does not always exist, and I think this one scales better than having a try and error environment around it.
I am open to better suggestions!

**Pull request closed without merging**